doc/api/v8.md

```diff
@@ -242,6 +242,62 @@ buffers and external strings.
 }
 ```
 
+## `v8.queryObjects(ctor[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `ctor` {Function} The constructor that can be used to search on the
```

Comment:
This currently takes a Function to match the Chromium DevTools console API, though IMO it would also be useful to offer a complementary API that allows users to register an object for filtering, and then return the registered objects in `v8.queryObjects` - this can be implemented based on e.g. symbols. It's worth noting that the operations allowed in the filter in the V8 API can be quite limited (I've seen crashes before when I tried to use a symbol-based approach when I was working on https://github.com/nodejs/node/pull/50572), so for this PR I think we should just go with the constructor-based API, since it's is supported by Chromium, so the prototype chain walking is more guaranteed to be safe.